tasks/request_files/test/unit_tests/test_request_files.py

```diff
@@ -1397,7 +1403,7 @@ def test_task_no_expire_days_env_var(
 
         mock_s3_cli = mock_boto3_client("s3")
         # mock_s3_cli.head_object = Mock()  # todo: Look into why this line was in so many tests without asserts.
-        mock_s3_cli.restore_object.side_effect = [None]
+        mock_s3_cli.restore_object.side_effect = [{"ResponseMetadata": {"HTTPStatusCode": 200}}]
```

Comment:
The status value should be 202 on an initial restore. 200 occurs if the granule has already been restored.

Reply:
Fixed